config/config.go

```diff
@@ -199,22 +201,50 @@ func get(args []string, l *logrus.Entry) (cfg *Config, configFileF string, err e
 			&cfg.Paths.AzureExporter:    "azure_exporter",
 			&cfg.Paths.VMAgent:          "vmagent",
 			&cfg.Paths.TempDir:          os.TempDir(),
-			&cfg.Paths.PTSummary:        "/usr/local/percona/pmm2/tools/pt-summary",
-			&cfg.Paths.PTPgSummary:      "/usr/local/percona/pmm2/tools/pt-pg-summary",
-			&cfg.Paths.PTMongoDBSummary: "/usr/local/percona/pmm2/tools/pt-mongodb-summary",
-			&cfg.Paths.PTMySqlSummary:   "/usr/local/percona/pmm2/tools/pt-mysql-summary",
+			&cfg.Paths.PTSummary:        "tools/pt-summary",
+			&cfg.Paths.PTPgSummary:      "tools/pt-pg-summary",
+			&cfg.Paths.PTMongoDBSummary: "tools/pt-mongodb-summary",
+			&cfg.Paths.PTMySqlSummary:   "tools/pt-mysql-summary",
 		} {
 			if *sp == "" {
 				*sp = v
 			}
 		}
 
+		switch {
+		case cfg.Paths.PathsBase == "" && cfg.Paths.ExportersBase == "":
+			cfg.Paths.PathsBase = pathBaseDefault
+			cfg.Paths.ExportersBase = filepath.Join(cfg.Paths.PathsBase, "exporters")
+		case cfg.Paths.PathsBase != "" && cfg.Paths.ExportersBase == "":
+			cfg.Paths.ExportersBase = filepath.Join(cfg.Paths.PathsBase, "exporters")
+		case cfg.Paths.PathsBase == "" && cfg.Paths.ExportersBase != "":
+			cfg.Paths.PathsBase = pathBaseDefault
```

Comment:
```suggestion
			cfg.Paths.PathsBase = cfg.Paths.ExportersBase
```

Reply:
Check this thread: https://perconacorp.slack.com/archives/C01KDTZ0G2C/p1627981201010800?thread_ts=1627895768.005500&cid=C01KDTZ0G2C